sdks/python/container/build.gradle

```diff
@@ -42,11 +42,75 @@ tasks.register("buildAll") {
   dependsOn ':sdks:python:container:py311:docker'
 }
 
-tasks.register("pushAll") {
+tasks.register("push38") {
   dependsOn ':sdks:python:container:py38:dockerPush'
+
+  doLast {
+    if (project.hasProperty("prune-images")) {
+      exec {
+        executable("docker")
+        args("system", "prune", "-a", "--force")
+      }
+    }
+  }
+}
+
+tasks.register("push39") {
+  // Enforce ordering to allow the prune step to happen between runs.
```

Comment:
this is a great idea. Can we consolidate this part in a loop like

```
for(i=0; i<versions.length; ++i) {
  taskName = "push" +versions[i]
  tasks.register(taskName) {
    if (i>0) {
        mustRunAfter(":sdks:python:container:push"+versions[i-1])
    } 
   ...
  }
}
```

this would make new version support / drop old version easier

Reply:
Done! Lets confirm that it works with the Actions build before merging though.

Build: https://github.com/damccorm/beam/actions/runs/5417461717
Dockerhub repos: https://hub.docker.com/repositories/damccorm (not sure if that will be visible)

Reply:
Update - it worked :)

Reply:
yes, the hub.docker page is visible